webplugin/js/app/zendesk-chat-service.js

```diff
@@ -322,9 +322,11 @@ var onTabClickedHandlerForZendeskConversations = function (event) {
         var assigneeInfo = currentGroupData && currentGroupData.users && Object.values(currentGroupData.users).find(function (member) {
             return member.userId == currentGroupData.metadata.CONVERSATION_ASSIGNEE
         })
+        if (document.getElementById('mck-tab-title').innerHTML == "Conversations" && assigneeInfo.role != KommunicateConstants.GROUP_ROLE.MODERATOR_OR_BOT) {
```

Comment:
in case if the customer is using some other language, will this == "Conversations" still work?

Reply:
But do we use translations on widget (like we do on Dashboard)? How can it get translated?

Reply:
check this https://docs.kommunicate.io/docs/web-localization#chat-localization-setup

Reply:
Updated. 
Picking the value dynamically now. instead of hardcoding